extensions/interactions/LogicProof/LogicProof.js

```diff
@@ -49,22 +49,38 @@ oppia.directive('oppiaInteractiveLogicProof', [
         );
         $scope.questionData.language.operators = $scope.typing[0].operators;
 
-        $scope.assumptionsString = logicProofShared.displayExpressionArray(
-          $scope.questionData.assumptions,
-          $scope.questionData.language.operators);
-        $scope.targetString = logicProofShared.displayExpression(
-          $scope.questionData.results[0],
-          $scope.questionData.language.operators);
+        $scope.displayExpression = function(exp) {
```

Comment:
I don't quite understand what this does. Can you add an explanatory comment?

Also, I would call the argument 'expression' -- when I first read 'exp' I thought it stood for 'exploration'.